src/vcu/src/controls.c

```diff
@@ -100,6 +107,10 @@ void execute_controls(void) {
       min_sensor_torque = temp_limited_torque;
     }
 
+    if (power_limited_torque < min_sensor_torque) {
```

Comment:
You may want to add a boolean to easily toggle this on and off, but not necessary right now